autopilot/README.md

```diff
@@ -1,11 +1,37 @@
 ## `autopilot` node
 The `autopilot` consists of control loops that actuate on setpoints that have been set by the mission control.
 
-These setpoints are:
-- **Attitude Servo**. Roll, pitch, yaw and speed targets.
-- **Altitude Heading**. Altitude, heading and speed targets.
-- **Depth Heading**. Depth, heading and speed targets.
-- **Fix Rudder**. Rudder, depth and speed targets.
-- **Waypoint**. Altitude, latitude, longitude and speed targets.
-
-The `autopilot` node checks that the `mission_control` node is running. If it detects that the `mission_control` node crashes, it sets the thruster velocity to 0 RPM and the fins to surface.
+### Subscribed Topics
+
+*/mngr/attitude_servo* ([mission_control/AttitudeServo](../mission_control/msg/AttitudeServo.msg))  
+Attitude servo setpoint.
+
+*/mngr/altitude_heading* ([mission_control/AltitudeHeading](../mission_control/msg/AltitudeHeading.msg))  
+Altitude heading setpoint.
+
+*/mngr/depth_heading* ([mission_control/DepthHeading](../mission_control/msg/DepthHeading.msg))  
+Depth heading setpoint.
+
+*/mngr/fixed_rudder* ([mission_control/FixedRudder](../mission_control/msg/FixedRudder.msg))  
+Fix rudder setpoint.
+
+*/mngr/waypoint* ([mission_control/Waypoint](../mission_control/msg/Waypoint.msg))  
+Waypoint setpoint.
+
+*/mngr/report_heartbeat* ([/mission_contro/ReportHeartbeat](../mission_control/msg/ReportHeartbeat.msg))
+The `autopilot` node receives on this topic regular pings from the `mission_control`. If the `autopilot` node doesn't receive the message form the `mission_control` node, it sets the thruster velocity to 0 RPM and the fins to surface.
+
+*/mngr/report_mission_execute_state* ([/mission_control/report_mission_execute_state](../mission_control/msg/ReportExecuteMissionState.msg))  
+
+The `autopilot` node receives on this topic the status of the mission. If the mission is complete, the `autopilot` node sets the thruster velocity to 0 RPM and the fins to surface.
+
+*/state* ([auv_interfaces/StateStamped](../auv_interfaces/msg/State.msg))  
+A full description of the vehicle's kinematic and dynamic state. It includes maneuvering and sea-keeping ship motion information as well as the vehicle's global position.
+
+### Published Topics
+
+*/input/autopilot/set_angles* ([fin_control/SetAngles](../fin_control/msg/SetAngles.msg))  
+Sends the fins angle position to the [cmd_actuator_mux](../cmd_actuators_mux/README.md) node.
```

Comment:
@patapufate Also, since topics are typically used to communicate state, one tends to drop the verb (a topic doesn't send or do anything):
```suggestion
*/input/autopilot/set_angles* ([fin_control/SetAngles](../fin_control/msg/SetAngles.msg))  
Fins' angle positions command, subscribed by the [cmd_actuator_mux](../cmd_actuators_mux/README.md) node.
```
Same elsewhere.

Reply:
Done